### PR TITLE
Add simple script to build wheels for a distribution

### PR DESCRIPTION
**Pull request by ilevkivskyi**
This builds on top of PR https://github.com/python/typeshed/pull/4259

### CODE REVIEW

**Comment by JukkaL** [comment]
I think that it would be simpler if the script would take an output directory argument, instead of creating a temp directory where to store the generated wheel. If a script wants to use this, it will have to parse the output to find the location of the generated wheel.

Also, the name of script is potentially misleading. Something like `generate_types_wheel.py` could be clearer?

File: scripts/generate_setup.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                # Packages there should be copied using a separate call to this
+                # function.
+                continue
+            else:
+                stub_dir = os.path.join(dst, entry + suffix)
+                shutil.copytree(os.path.join(base_dir, entry), stub_dir)
+
+        # We add original METADATA file in case some type-checking tool will want
+        # to use it. Note that since it is not easy to package it outside of a package,
+        # we copy it to every package in given distribution.
```
Comment (JukkaL) [inline scripts/generate_setup.py:96]: METADATA.toml is needed to decide whether the stubs support Python 2/3, in particular.

**Comment by JelleZijlstra** [comment]
Any update on this PR?

**Comment by ilevkivskyi** [comment]
This will be added to a separate repo with a more restricted write access (for security reasons).

**Pull request closed without merging**